torch_geometric/utils/hetero.py

```diff
@@ -119,11 +122,11 @@ def construct_bipartite_edge_index(
         edge_attr = torch.cat(edge_attrs, dim=0)
 
     if is_sparse_tensor:
-        # TODO Add support for `SparseTensor.sparse_sizes()`.
         edge_index = SparseTensor(
-            row=edge_index[1],
-            col=edge_index[0],
+            row=edge_index[0],
+            col=edge_index[1],
             value=edge_attr,
-        )
-
+            sparse_sizes=n_nodes,
+        ).t()
```

Comment:
I'm not too familiar with `torch_sparse.SparseTensor`, but do we want to avoid calling `t()` to preserve contiguity of the tensor? @rusty1s

Reply:
(Has been a while so this may already be resolved but) Changed the number of edges in `test_hgt_conv.py`, which leads to the following error in master but passes with the edit.
```
====================================== short test summary info ======================================
FAILED test/nn/conv/test_hgt_conv.py::test_hgt_conv_same_dimensions - ValueError: Encountered tensor with size 10 in dimension 0, but expected size 6.
```